buildSrc/build.gradle

```diff
@@ -58,6 +58,7 @@ dependencies {
 
     // Provided/Optional Dependencies
     compile 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
+    implementation 'org.apache.logging.log4j:log4j-core:2.11.1'
```

Comment:
Hope this doesn't affect something.
Had to add it because of the following error:
```
> Task :buildSrc:compileGroovy
error: Bad service configuration file, or exception thrown while constructing Processor object: javax.annotation.processing.Processor: Provider org.apache.logging.log4j.core.config.plugins.processor.PluginProcessor not found
startup failed:
Compilation failed; see the compiler error output for details.
```